netty/src/main/java/io/grpc/netty/NettyClientStream.java

```diff
@@ -50,12 +50,25 @@
 import io.netty.handler.codec.http2.Http2Stream;
 import io.netty.util.AsciiString;
 
+import java.util.IdentityHashMap;
+import java.util.Map;
+import java.util.function.Supplier;
```

Comment:
That's Java 8: https://docs.oracle.com/javase/8/docs/api/java/util/function/Supplier.html